src/commands/new.ts

```diff
@@ -799,24 +799,21 @@ module.exports = {
       stopSpinner(" Configuring app.json", "⚙️")
       // #endregion
 
-      // #region Run Format
+      // #region Run Prebuild
       // we can't run this option if we didn't install deps
       if (installDeps === true) {
         // Check if we need to run prebuild to generate native dirs based on workflow
         // Prebuild also handles the packager install
-        const prebuildMessage = ` Generating native template via Expo Prebuild`
```

Comment:
Hm I think some of these spaces were intended to make the terminal output look nicer, let's see if things don't looked mashed up against the spinner / icon bullet

Reply:
Oh yeah I removed that because it looked odd to me: I'll try to dig up screenshots to compare with

Reply:
What I observed that led to me editing out the space:

<img width="470" alt="Screenshot 2024-08-27 at 4 07 12 PM" src="https://github.com/user-attachments/assets/e99ca163-4a38-4b51-b509-5dc7c1ef5c73">

What it looks like now on this branch (didn't realize the MST icon had changed):
<img width="432" alt="Screenshot 2024-09-05 at 2 41 16 PM" src="https://github.com/user-attachments/assets/45d2a619-256e-4b01-9780-540a32670760">

I notice I missed space for "Configuring app.json", so that's still inconsistent.

What was the intended design here?

Reply:
Ok cool, no real intended design just have them consistent, it looks fine without the space. It must have just been left over from elsewhere at some point

Reply:
👍 I'll remove the one last remaining space then